game_logic/src/cg/group4/util/timer/Timer.java

```diff
@@ -272,7 +272,7 @@ public final Subject getTickSubject() {
          * Length definition of one stroll.
          * (5 * 60 seconds = 5 minutes)
          */
-        STROLL(5 * 60),
+        STROLL(2 * 60),
```

Comment:
The stroll now lasts 120 seconds, so either comment is outdated or it should be changed back to 300 seconds.